torchok/data/datasets/classification/classification.py

```diff
@@ -12,6 +12,33 @@
 from torchok.data.datasets.base import ImageDataset
 
 
+def process_multilabel(labels: str, num_classes: int) -> np.array:
```

Comment:
return type is `np.ndarray`. Why it is not static method of classification dataset?

Reply:
Because it may be used in other datasets that do not inherit classification dataset

Reply:
change return type to `np.ndarray`

Reply:
Changed